src/status_im/contexts/onboarding/events.cljs

```diff
@@ -73,7 +73,7 @@
                                       :onboarding/navigated-to-enter-seed-phrase-from-screen
                                       :screen/onboarding.new-to-status)]]
                :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
-                                 :dispatch [:onboarding/navigate-to-identifiers]}]
+                                 :dispatch [:onboarding/navigate-to-enable-notifications]}]
```

Comment:
@jo-mut are you sure it will work? did you test it? i believe navigate within stack will be broken

Reply:
@flexsurfer yes, i tested it and works just alright

Reply:
I didn't tested, but back button on enable notifications might be broken :thinking:

btw, do we need back button in that screen?

Reply:
yes, just tried back button to Keys saved screen, after which we gets stuck on that.

Reply:
its not broken, works fine. I have tested it several times. The code was updated in this pr https://github.com/status-im/status-mobile/pull/20386https://github.com/status-im/status-mobile/pull/20386 and this allows this event `:onboarding/navigate-to-enable-notifications` to works just fine

Reply:
ooh yeah from the `enable notifications screen` I think we dont need the navigate back from in this screen

Reply:
I think we can remove the navigate back button from this screen. Users would navigate to the Identifiers screen before but since that screen is not present we dont need to navigate back from `Enable Notifications` screen

cc @cammellos

Reply:
if there's no screen that we would go back to (i.e it's the first screen post login), then we should remove the back button, yes